api/src/main/java/org/openmrs/scheduler/timer/TimerSchedulerServiceImpl.java

```diff
@@ -542,5 +542,13 @@ public void scheduleIfNotRunning(TaskDefinition taskDef) {
 			}
 		}
 	}
+
+	@Override
+	public TaskDefinition getTaskByUuid(String uuid) {
+		if (log.isDebugEnabled()) {
```

Comment:
why this check is needed? `log.debug(...)` will only log if debug is enabled anyway. I'm just curious.

Reply:
logging directly brings about string concatenation which is abit expensive for the application in terms of its performance thats why under this code snippet we only log a message when the logger's debug mode is enabled

Reply:
Do we even need to log accessing a task by uuid?

Reply:
@dkayiwa i used logging a task by uuid because of the unique identification the uuid  brings for the task defined in the task definition but i will go ahead and remove the logging if its not appropriate here

Reply:
@dkayiwa i have removed the logger,,kindly requesting for your review 

Thanks